### PR TITLE
#212: m2 repo settings as MAVEN_ARGS 

### PR DESCRIPTION
**Pull request by mvomiero**
Fixes: #212

### CODE REVIEW

**Review by VinceHeu — APPROVED** [review]
Thanks for your addition. 
Looks great :)

**Review by hohwille — APPROVED** [review]
@mvomiero thanks for your PR. You implemented the correct solution. 👍 
I left a small comment for improvement of robustness to avoid NullPointerException if called from outside of the project (IDE_HOME).
Please add a null check and then I can merge. Thanks.

File: cli/src/main/java/com/devonfw/tools/ide/variable/IdeVariables.java [hunk 1]
```diff
@@ -43,6 +41,10 @@ public interface IdeVariables {
   /** {@link VariableDefinition} for version of maven (mvn). */
   VariableDefinitionVersion MVN_VERSION = new VariableDefinitionVersion("MVN_VERSION", "MAVEN_VERSION");
 
+  /** {@link VariableDefinition} arguments for maven to set the m2 repo location. */
+  VariableDefinitionString MAVEN_ARGS = new VariableDefinitionString("MAVEN_ARGS", null, c -> "-s " + c.getIdeHome().resolve("conf/.m2/settings.xml"), false,
+      true);
```
[the remark this answers is not in the document]
Comment (mvomiero) [reply cli/src/main/java/com/devonfw/tools/ide/variable/IdeVariables.java]: True! thank you for noticing the problem!
Implemented in a `getMavenArgs()` method in `IdeContext`

**Review by hohwille — APPROVED** [review]
@mvomiero thanks for the update. 👍 
Ready for merge.

**Pull request merged**